packages/lbd/PKGBUILD

```diff
@@ -3,18 +3,17 @@
 
 pkgname=lbd
 pkgver=20130719
-pkgrel=5
+pkgrel=6
 groups=('blackarch' 'blackarch-recon')
 pkgdesc='Load Balancing detector,'
 url='http://ge.mine.nu/code/lbd'
 arch=('any')
 license=('GPL')
-depends=('bash' 'gnu-netcat')
+depends=('bash' 'dnsutils' 'openbsd-netcat')
 #source=('http://ge.mine.nu/code/lbd')
 source=("$pkgname")
 sha512sums=('2436f9e372af7281a1e3dae1ab9d020933d35277664783fbc39736630a8c8d8acbecb2b5028a593e465ca4729a990daacdd86191570ccdaab912a8021a61a21f')
 
 package() {
   install -Dm 755 $pkgname "$pkgdir/usr/bin/$pkgname"
 }
-
```

Comment:
Beware not to remove the ending blank line.

Reply:
Ok